cmd/context/create.go

```diff
@@ -265,6 +265,18 @@ func (c *ContextCommand) initOktetoContext(ctx context.Context, ctxOptions *Cont
 		cfg = kubeconfig.Create()
 	}
 	okteto.AddOktetoCredentialsToCfg(cfg, &userContext.Credentials, ctxOptions.Namespace, userContext.User.ID, okteto.Context().Name)
+
+	ktClient, err := okteto.NewKubeTokenClient(ctxOptions.Context, userContext.Credentials.Token, ctxOptions.Namespace)
+	if err != nil {
+		oktetoLog.Debug("unable to retrieve kubetoken client")
+		return err
+	}
+	if err := ktClient.Head(); err == nil {
+		okteto.AddKubetokenExecToCfg(cfg, userContext.User.ID)
```

Comment:
I think this has to be done only when `okteto kubeconfig` command is executed. If I'm not wrong, this would modify all the internal Kubeconfig for the CLI forcing users to have `okteto` in the path when the CLI is executed and it's something we don't want. We just want to force it when they use `kubectl` or any other external tool using our generated kubeconfig.

Also, there is intention to add a feature flag to control when to use kubetoken (https://github.com/okteto/app/issues/6886) and we should check it before doing it not

Reply:
ok, i think i understood then wrong wher to add the implementation :) i will review and make required modifications